include/networkit/generators/ChungLuGeneratorAlamEtAl.hpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * ChungLuGeneratorAlamEtAl.hpp
+ *
+ *  Created on: Dec 23, 2013
+ *      Author: Henning
+ *      Contributors: Hoske/Weisbarth/Hering
+ */
+
+#ifndef NETWORKIT_GENERATORS_CHUNG_LU_GENERATOR_ALAM_ET_AL_HPP_
+#define NETWORKIT_GENERATORS_CHUNG_LU_GENERATOR_ALAM_ET_AL_HPP_
+
+#include <networkit/generators/StaticDegreeSequenceGenerator.hpp>
+#include <networkit/generators/StaticGraphGenerator.hpp>
+
+namespace NetworKit {
+
+/**
+ * @ingroup generators
+ * Given an arbitrary degree sequence, the Chung-Lu generative model
+ * will produce a random graph with the same expected degree sequence.
+ *
+ * For reference of the concept of the original model see Chung, Lu: The average distances in random
+ * graphs with given expected degrees and Chung, Lu: Connected Components in Random Graphs with
+ * Given Expected Degree Sequences. Aiello, Chung, Lu: A Random Graph Model for Massive Graphs
+ * describes a different generative model which is basically asymptotically equivalent but produces
+ * multi-graphs.
+ *
+ * This implementation follows a newer parallelizable algorithm by M. Alam, M. Khan, A. Vullikanti
+ * and M. Marathe, "An Efficient and Scalable Algorithmic Method for Generating Large-Scale Random
+ * Graphs". https://ieeexplore.ieee.org/document/7877110 . The time complexity is O(m/p + delta +
```

Comment:
Use markdown style for URLS: 

```
["An Efficient and Scalable Algorithmic Method for Generating Large-Scale Random Graphs"](https://ieeexplore.ieee.org/document/7877110)
```

Best also add URLs to the other references.